sources/packages/web/src/types/contracts/StudentScholasticStanding.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Possible scholastic standing changes types reported by an institution when the
+ * program enrolled by the student was not completed as expected and the institution
+ * must report the change to the Ministry.
+ */
+export enum StudentScholasticStandingChangeType {
+  /**
+   * Student went from full-time to part-time studies (vice-versa) or started a different program.
+   * This scholastic standing change type is only used to finish the current application. The actual
+   * changing from full-time to part-time studies (vice-versa) is not part of the scholastic standing change.
+   */
+  ChangeInIntensity = "Change in intensity",
```

Comment:
Not part of the PR. But curious to know more about this type of Scholastic standing change @andrewsignori-aot  @JasonCTang

Reply:
its  student changing from full-time to part-time studies and (vice-versa)

Reply:
Thanks. I got that from comment, but my curiosity was to know when a student want to change from full-time to part time what happens on end to end? what happens to their tuition remittance and other costs involved related to the offering.?

Reply:
@dheepak-aot, from business perspective @JasonCTang can provide more details.
From a system perspective, depending on the type, it may impact the offering end date or its associated costs (see images below).
For any of those cases, we will clone the existing offering, change the end date and/or costs and execute the assessment again.
To be clear, independent of the description, the above-mentioned scholastic type is not about changing from full-time to part-time, it is just about reporting that the current application is being finished earlier.

![image](https://user-images.githubusercontent.com/61259237/225791005-5a9ed10e-a20f-46a4-b600-80ffe66fe6f0.png)

![image](https://user-images.githubusercontent.com/61259237/225791059-6ca46bac-ec61-4b1d-b57f-02880cbde0ee.png)